engine.py

```diff
@@ -429,8 +429,10 @@ def update_image():
         # launch the denoise process if turned on
         if self.rm.do_denoise and not isProblem:
             base, ext = render_output.rsplit('.', 1)
+            denoise_options = []
```

Comment:
btw the short form of this is very clean in python:
denoise_options = ["-t%d" %self.rm.threads] if self.rm.threads != 0 else []